### PR TITLE
Updated command for helloworld sample

### PR DESCRIPTION
**Pull request by jatanloya**
It seems that the current installation has newer directory name and enclave name for running on Ubuntu 18.04 in simulation mode.

### CODE REVIEW

**Review by hani1i1 — APPROVED** [review]

File: samples/helloworld/Makefile
Comment (jatanloya) [reply samples/helloworld/Makefile:20]: Yes, that is the intention. I've updated the Makefile in `host` and `enclave` directories.

**Comment by mingweishih** [comment]
bors try

File: samples/helloworld/enclave/Makefile
Comment (mingweishih) [inline samples/helloworld/enclave/Makefile:39]: Also need to update here.

Comment (jatanloya) [reply samples/helloworld/enclave/Makefile]: Updated

**Comment by mingweishih** [comment]
bors try

**Comment by mingweishih** [comment]
I'll wait for the CI to pass first. In the meantime, can you squash and signoff the commits? Thanks for the contribution!

**Comment by jatanloya** [comment]
> I'll wait for the CI to pass first. In the meantime, can you squash and signoff the commits? Thanks for the contribution!

Yes done, sorry had missed it earlier! Thanks!

**Comment by mingweishih** [comment]
LGTM. Can you squash commits (making them into a single one) before we can merge? Thanks!

**Review by mingweishih — APPROVED** [review]
Thanks for the contribution. See my earlier comments before we can merge the PR.

**Comment by jatanloya** [comment]
Thanks! I've squash the commits into one.

**Comment by mingweishih** [comment]
It seems you accidentally introduce other changes. Can you double check (i.e., making sure your branch includes only necessary changes compared to the latest master branch)?

**Comment by mingweishih** [comment]
I'd recommend renaming your branch, creating a new branch (with the original name) based off the master, apply your change, and do a force push.

**Comment by jatanloya** [comment]
Would it be okay to make a new fork and PR?

**Comment by mingweishih** [comment]
sure

**Pull request closed without merging**
